.github/workflows/build-test-package.yml

```diff
@@ -134,9 +134,9 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [36, 37, 38, 39]
+        python-version: [37, 38, 39, 310]
         include:
-          - itk-python-git-tag: "v5.2.1.post1"
+          - itk-python-git-tag: "v5.3rc03"
```

Comment:
Should these be moved to `env` variables? See [https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L5](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L5)

Reply:
Yes -- these updates are based on ITKModuleTemplate `main`. Could you please port the ITKSplitComponents updates to ITKModuleTemplate?

Reply:
Please see [https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126).